pkgs/games/steam/chrootenv.nix

```diff
@@ -2,6 +2,7 @@
 , steam-runtime, steam-runtime-i686 ? null
 , withJava ? false
 , withPrimus ? false
+, extraPkgs ? [ ] # extra packages to add to targetPkgs and multiPkgs
```

Comment:
Please make this a function:

`extraPkgs ? pkgs: []`

Also, they actually would be added only to `targetPkgs`.